app/views/shared/_banner.html.erb

```diff
@@ -21,7 +21,7 @@
           </button>
         </div>
       </header>
-      <div class="usa-banner__content usa-accordion__content hide" id="gov-banner">
+      <div class="usa-banner__content usa-accordion__content" id="gov-banner">
```

Comment:
Since we're removing the `.hide` class, do we need to add in `hidden=true`? Or is that handled by the new JS

Reply:
> Since we're removing the `.hide` class, do we need to add in `hidden=true`? Or is that handled by the new JS

Good question. I wondered this as well. The official USWDS examples omit `hidden` in the static markup:

https://designsystem.digital.gov/components/banner/

As you suspected, it's added back in the accordion initialization:

https://github.com/uswds/uswds/blob/f7acaf7/src/js/components/accordion.js#L86-L91

I expect it could be done this way intentionally so that in a browser where JavaScript is disabled, the expanded contents of the banner are available to be read.

Reply:
>https://designsystem.digital.gov/components/banner/
>
>[...] I expect it could be done this way intentionally so that in a browser where JavaScript is disabled, the expanded contents of the banner are available to be read.

Ah, looking closer, it's even outlined very explicitly under the Accessibility guidance:

>**The banner's accordion uses javascript to set the `aria-hidden` value of its content area.** To ensure that your content is accessible in the event that the JavaScript does not load or is disabled, you should not `set aria-hidden="true"` on usa-banner__content.

Reply:
Ok sweet, thanks for digging in!